lib/ansible/modules/network/fortimanager/fmgr_secprof_web.py

```diff
@@ -0,0 +1,1236 @@
+
+  ftgd_wf_ovrd:
+    description:
+      - Allow web filter profile overrides.
+    required: false
+
+  ftgd_wf_rate_crl_urls:
+    description:
+      - Enable/disable rating CRL by URL.
+      - choice | disable | Disable rating CRL by URL.
+      - choice | enable | Enable rating CRL by URL.
+    required: false
+    choices: ["disable", "enable"]
+
+  ftgd_wf_rate_css_urls:
+    description:
+      - Enable/disable rating CSS by URL.
+      - choice | disable | Disable rating CSS by URL.
+      - choice | enable | Enable rating CSS by URL.
+    required: false
+    choices: ["disable", "enable"]
+
+  ftgd_wf_rate_image_urls:
+    description:
+      - Enable/disable rating images by URL.
+      - choice | disable | Disable rating images by URL (blocked images are replaced with blanks).
+      - choice | enable | Enable rating images by URL (blocked images are replaced with blanks).
+    required: false
+    choices: ["disable", "enable"]
+
+  ftgd_wf_rate_javascript_urls:
+    description:
+      - Enable/disable rating JavaScript by URL.
+      - choice | disable | Disable rating JavaScript by URL.
+      - choice | enable | Enable rating JavaScript by URL.
+    required: false
+    choices: ["disable", "enable"]
+
+  ftgd_wf_filters_action:
+    description:
+      - Action to take for matches.
+      - choice | block | Block access.
+      - choice | monitor | Allow access while logging the action.
+      - choice | warning | Allow access after warning the user.
+      - choice | authenticate | Authenticate user before allowing access.
+    required: false
+    choices: ["block", "monitor", "warning", "authenticate"]
+
+  ftgd_wf_filters_auth_usr_grp:
+    description:
+      - Groups with permission to authenticate.
+    required: false
+
+  ftgd_wf_filters_category:
+    description:
+      - Categories and groups the filter examines.
+    required: false
+
+  ftgd_wf_filters_log:
+    description:
+      - Enable/disable logging.
+      - choice | disable | Disable setting.
+      - choice | enable | Enable setting.
+    required: false
+    choices: ["disable", "enable"]
+
+  ftgd_wf_filters_override_replacemsg:
+    description:
+      - Override replacement message.
+    required: false
+
+  ftgd_wf_filters_warn_duration:
+    description:
+      - Duration of warnings.
+    required: false
+
+  ftgd_wf_filters_warning_duration_type:
+    description:
+      - Re-display warning after closing browser or after a timeout.
+      - choice | session | After session ends.
+      - choice | timeout | After timeout occurs.
+    required: false
+    choices: ["session", "timeout"]
+
+  ftgd_wf_filters_warning_prompt:
+    description:
+      - Warning prompts in each category or each domain.
+      - choice | per-domain | Per-domain warnings.
+      - choice | per-category | Per-category warnings.
+    required: false
+    choices: ["per-domain", "per-category"]
```

Comment:
To be consistent with other Ansible modules kindly modify the choices options at all required places as:
```suggestion
    choices: 
    - per-domain
    - per-category
```